canonical/ubuntu.com
# [WD-7724] chiselled Ubuntu chart and table

## Done

- Implemented a table and a chart on /containers/chiselled Chiselled Ubuntu landing page according to [design in Figma](https://www.figma.com/file/YfNc8KmYqa2e7JgbFo6lXa/24.04-U.com---%2Fcontainers?type=design&node-id=111-5289&mode=design&t=4cx8gEHI2Oe2pxgU-4)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /containers/chiselled and scroll down to "Production-ready, ultra-small Ubuntu containers" section where the table and a chart are located. Check if they look the same as in the design. 
   - also check if the table and a chart look the as in design for mobile and tablet screen resolutions. 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7724

![Screenshot 2023-12-08 at 11 04 32](https://github.com/canonical/ubuntu.com/assets/15943863/b0be1bc6-6970-4e89-8743-9278d9b258a7)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
